run_tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -ex
-isort --check --diff ghstack
-black --check ghstack
+export LIBCST_PARSER_TYPE=native
```

Comment:
What's the reasoning for this? How come not the default?

Reply:
The native (rust) parser is faster and supports 3.10+ syntax like match statements. I'm planning to make that choice the default in upstream usort, so it's not necessary, and I can drop it from the PR if you prefer.